vertx-template-engines/vertx-web-templ-jte/pom.xml

```diff
@@ -58,7 +58,24 @@
           </execution>
         </executions>
       </plugin>
-
+      <plugin>
+        <groupId>gg.jte</groupId>
+        <artifactId>jte-maven-plugin</artifactId>
+        <version>${jte.version}</version>
+        <configuration>
+          <sourceDirectory>src/test/templates</sourceDirectory>
+          <targetDirectory>target/test-classes</targetDirectory>
```

Comment:
I didn't found a way to split the generated `.java` from the `.class` so both end up in the classpath.

Reply:
Oh, right! I think I could make this configurable. If I understand you correct, you'd like to have a third parameter, that specifies where the .java files go to, right?

Reply:
yes, say for example, I'd like to have the sources being saved in `target/generated-test-sources` and the final classes in `target/test-classes` (since this is for testing it's ok to have it together but for prod, it would mean i would only package `.class` files in the final jar